man/udev.xml

```diff
@@ -68,9 +68,10 @@
       pointing to the device node, or run a specified program as part of
       the event handling.</para>
 
-      <para>A rule consists of a comma-separated list of one or more key-value pairs.
-      Each key has a distinct operation, depending on the used operator. Valid
-      operators are:</para>
+      <para>A rule consists of a comma-separated list of one or more key-operator-value pairs.
+      Each key has a distinct operation, depending on the used operator.</para>
+
+      <refsect2><title>Operators</title>
       <variablelist>
```

Comment:
Added refsect2's as I find it fairly difficult to find stuff.
Also did not indent the lines within them (not sure if it will make reviewing more difficult)

Reply:
Yes, please reindent stuff. For reviewing, we can ignore whitespace changes.

`<title>` should be on a separate line after `<refsect*>`.

Reply:
Didn't think it was possible on github. Cool!
![image](https://user-images.githubusercontent.com/4318379/97591697-d5a94e00-1a3a-11eb-858b-1e27a6f50336.png)